cwms-data-api/src/main/java/cwms/cda/api/TextTimeSeriesController.java

```diff
@@ -0,0 +1,357 @@
+            logger.log(Level.SEVERE, re.toString(), e);
+            ctx.status(HttpServletResponse.SC_INTERNAL_SERVER_ERROR).json(re);
+        }
+    }
+
+
+    @OpenApi(
+        description = "Deletes requested text timeseries id",
+        pathParams = {
+            @OpenApiParam(name = TS_ID, description = "The time series identifier to be deleted"),
+        },
+        queryParams = {
+                @OpenApiParam(name = OFFICE, required = true, description = "Specifies the "
+                        + "owning office of the timeseries identifier to be deleted"),
+                @OpenApiParam(name = Controllers.TEXT_MASK, required = true, description = "The "
+                        + "standard text pattern to match. "
+                        + "Use glob-style wildcard characters instead of sql-style wildcard "
+                        + "characters for pattern matching."
+                        + "  For StandardTextTimeSeries this should be the Standard_Text_Id (such"
+                        + " as 'E' for ESTIMATED)"),
+                @OpenApiParam(name = "mode", required = true, type =
+                        TimeSeriesTextMode.class,
+                        description = "Type of delete to perform. Options are:\n"
+                                + "ALL\n"
+                                + " - Delete Standard and Regular text timeseries values for the "
+                                + "specified time series.\n"
+                                + "STANDARD\n"
+                                + " - Delete Standard text time series for the specified text "
+                                + "timeseries.\n"
+                                + "REGULAR\n"
+                                + " - Delete Regular text timeseries values for the specified "
+                                + "time series.\n"
+                ),
+                @OpenApiParam(name = TIMEZONE, description = "Specifies "
+                        + "the time zone of the values of the begin and end fields (unless "
+                        + "otherwise specified). If this field is not specified, "
+                        + "the default time zone of UTC shall be used."),
+                @OpenApiParam(name = START, required = true, description = "The start of the time"
+                        + " window"),
+                @OpenApiParam(name = END, description = "The end of the time window. If specified"
+                        + " the text associated with all times from start to end (inclusive) is "
+                        + "deleted."),
+                @OpenApiParam(name = VERSION_DATE, description = "The version date for the time "
+                        + "series.  If not specified, the minimum or maximum version date "
+                        + "(depending on p_max_version) is used."),
+                @OpenApiParam(name = MAX_VERSION, type = Boolean.class, description = "Whether to"
```

Comment:
`MAX_VERSION` is confusing as I would think the default behavior would be true. I'd recommend leaving this parameter out and always passing in true to the pl/sql if the `VERSION_DATE` is null.